client/AccountsClient.ts

```diff
@@ -44,14 +39,15 @@ export default class AccountsClient {
     }
 
     private readonly _endpoint: string;
-    private readonly _defaultBehavior: RequestBehavior;
+    private readonly _defaultBehavior: RequestBehavior<DB>;
     private readonly _iframeBehavior: IFrameRequestBehavior;
     private readonly _redirectClient: RedirectRpcClient;
 
-    constructor(endpoint: string = AccountsClient.DEFAULT_ENDPOINT, defaultBehavior?: RequestBehavior) {
+    constructor(endpoint: string = AccountsClient.DEFAULT_ENDPOINT,
+                defaultBehavior?: RequestBehavior<DB>) {
         this._endpoint = endpoint;
         this._defaultBehavior = defaultBehavior || new PopupRequestBehavior(
-            `left=${window.innerWidth / 2 - 400},top=75,width=800,height=850,location=yes,dependent=yes`);
+            `left=${window.innerWidth / 2 - 400},top=75,width=800,height=850,location=yes,dependent=yes`) as any;
```

Comment:
Why not rather properly type the options in the `PopupRequestBehavior` as `string`?

Reply:
It's not the options I type as any, but the created object. It's strange I have to do it, but otherwise the compiler complains about something like `B (extends BehaviorType) not assignable to DB (extends BehaviorType)', yet we know that everything works fine...